packages/gestalt/src/FlyoutUtils/__tests__/Controller.test.js

```diff
@@ -1,7 +1,4 @@
 // @flow
-/* eslint import/imports-first: 0 */
-jest.unmock('../Controller');
```

Comment:
I started to move this down when I turned the linter on and realized its not needed for this test. I am removing it here because otherwise I would need to move it below the imports